blog/7-streampark-usercase-haibo.md

```diff
@@ -1,103 +1,103 @@
 ---
 slug: streampark-usercase-haibo
-title: StreamPark 一站式计算利器在海博科技的生产实践，助力智慧城市建设
-tags: [StreamPark, 生产实践, FlinkSQL]
+title: StreamPark An All-in-One Computation Tool in Haibo Tech's Production and its Role in Smart City Construction
```

Comment:
An All-in-One Computation Tool in Haibo Tech's Production Practice and facilitation in Smart City Construction